PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitJobUploadStorageModel.kt

```diff
@@ -52,5 +53,7 @@ class SparkSubmitJobUploadStorageModel {
     @Attribute("storage_account_type")
     var storageAccountType: SparkSubmitStorageType = SparkSubmitStorageType.DEFAULT_STORAGE_ACCOUNT
 
-    @Transient var errorMsg: String? = SparkSubmissionJobUploadStoragePanel().notFinishCheckMessage
+    @get:Transient @set:Transient var errorMsg: String? = null
+
+    fun getCredentialAzureBlobAccount(): String? = storageAccount?.let{ SERVICE_NAME_PREFIX + storageAccount }
```

Comment:
This method will be called when we save or load the storage key. We only save the storage key after storage account and storage key are validated and proved to be legal. We will load the storage key when we open run configuration dialog the first time and storage account can be promised to be legal at that time (since they are saved legally).

---
In reply to: [226156308](https://github.com/Microsoft/azure-tools-for-java/pull/2220#discussion_r226156308) [](ancestors = 226156308)